presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java

```diff
@@ -27,7 +27,7 @@
         implements AutoCloseable
 {
     private static final SecureRandom random = new SecureRandom();
-    private static final int RANDOM_SUFFIX_LENGTH = 12;
+    private static final int RANDOM_SUFFIX_LENGTH = 6;
```

Comment:
about 2^31 options, but since we generate them a lot.... can this be eg 8?

we can shorted the table name's base a bit maybe?

Reply:
Currently, only `0startingwithdigit` is the problem. Let me rename to `0startwithdigit`. and drop the commit to shorten.